storage/interceptors/notifications_interceptor.go

```diff
@@ -82,18 +101,19 @@ func (ni *NotificationsInterceptor) OnTxUpdate(h storage.InterceptUpdateOnTxFunc
 		}
 		oldObject.SetLabels(nil)
 
-		// if the resource update contains change in the platform ID field this means that the notification would be processed by
-		// two platforms - one needs to perform a delete operation and the other needs to perform a create operation.
-		if oldPlatformID != updatedPlatformID {
-			if err := CreateNotification(ctx, repository, types.CREATED, updatedObject.GetType(), updatedPlatformID, &Payload{
+		for _, platformID := range addedPlatformIDs {
```

Comment:
I think the if above about removing the labels of the updated object should be below this for statement.
This is because if the supported platform is new, it does not know anything about this notification for now and it will need all labels for this object. This has to be double checked of course.

Reply:
Discussed.